pkg/kubelet/dockertools/manager.go

```diff
@@ -48,12 +48,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/network/hairpin"
 	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
Ill do this janitorial work later today
On May 19, 2016 2:27 PM, "Lantao Liu" <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/25688#discussion_r63954059>
> :
>
> > @@ -48,12 +48,12 @@ import (
> >  	"k8s.io/kubernetes/pkg/kubelet/network/hairpin"
> >  	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
> >  	"k8s.io/kubernetes/pkg/kubelet/qos"
> > -	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
>
> @sjpotter <https://github.com/sjpotter> @dchen1107
> <https://github.com/dchen1107> If so, we may also want to change
> kubecontainer and kubepod in the future. :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25688/files/d2700058430e71b4046bc1886fd99e808bee090f#r63954059>
>